src/resources/views/crud/inc/show_tabbed_table.blade.php

```diff
@@ -1,15 +1,16 @@
 @php
     $horizontalTabs = $crud->getTabsType()=='horizontal' ? true : false;
+    $columnsWithoutTab = $crud->columns();
```

Comment:
This are not really the `columns without tab` right ? 🦅 👁️

Reply:
I had alreayd changed that one... you were too quick for me 😆